README.md

```diff
@@ -128,18 +142,23 @@ animatedUi =
         }
 ```
 
-Then create animated `Element`s!
+Finally we need to wrap the Elm-UI `Element`s!
+Because you will likely still need to create non-animated
+`Element`s, lets give these a similar name `ael` and `acolumn`
+for "animated element" and "animated column". Just paste these into your file
 
 ```elm
-el : Animation -> List (Element.Attribute msg) -> Element msg -> Element msg
-el =
+ael : Animation -> List (Element.Attribute msg) -> Element msg -> Element msg
+ael =
```

Comment:
would you mind calling these `animatedEl` and `animatedColumn`? I feel it might be clearer for newcomers what they are

Reply:
Makes sense! Just made the changes!